docs/evaluation.md

```diff
@@ -384,6 +383,8 @@ First, as a preparatory step, introduce an auxilary step in our `qaFlow`
 example:
 
 ```js
+import { run } from '@genkit-ai/core';
+
```

Comment:
```suggestion
```

Reply:
`@genkit-ai/core` is not public API, must never be directly imported by end user

Reply:
use instead:
```ts
ai.run(...)
```